cmd/zrok/enable.go

```diff
@@ -2,6 +2,10 @@ package main
 
 import (
 	"fmt"
+	"os"
+	user2 "os/user"
+	"time"
+
```

Comment:
I'll fix this when I merge it... but you should see if you can adjust your vscode formatter to agree with the jetbrains stuff with regard to imports.